src/Pose_TEST.cc

```diff
@@ -155,3 +155,22 @@ TEST(PoseTest, OperatorStreamOut)
   EXPECT_EQ(stream.str(), "0.1 1.2 2.3 0 0.1 1");
 }
 
+/////////////////////////////////////////////////
+TEST(PoseTest, PoseElements)
+{
+  const math::Pose3i pose(0, 1, 2, 0, 0, 0);
+
+  int x = pose.Pos().X();
+  int y = pose.Pos().Y();
+  int z = pose.Pos().Z();
+  int Roll = pose.Rot().Roll();
+  int Pitch = pose.Rot().Pitch();
+  int Yaw = pose.Rot().Yaw();
```

Comment:
Small style detail: could you use lowercase for all variables? Such as:

```suggestion
  int yaw = pose.Rot().Yaw();
```

Reply:
ohh yeah, sure, I can do this. I've forget about the variable style code, sorry!